// tao_name/dist/theArtOfBecomingInvisible/star.js
/*
 * @Description:
 * @Version: 1.0.0
 * @Author: lax
 * @Date: 2023-05-15 23:15:22
 * @LastEditors: lax
 * @LastEditTime: 2024-05-18 20:59:08
 */
module.exports = {
  /**
   * 九星：九星源于古代天文中北斗七星，加上左辅星与右弼星一共九个星宿。
   * 北斗九星，七现二隐
   * 天枢/贪狼/天蓬（北斗一：Dubhe）
   * 天璇/巨门/天芮（北斗二：Merak）
   * 天玑/禄存/天冲（北斗三：Phecda）
   * 天权/文曲/天辅（北斗四：Megrez）
   * 玉衡/廉贞/天禽（北斗五：Alioth）
   * 开阳/武曲/天心（北斗六：Mizar）
   * 摇光/破军/天柱（北斗七：Alkaid）
   * 洞明/左辅/天任 (辅增一：MUA83,IQ)
   * 隐元/右弼/天英 (相星：5 CVn)
   */
  STAR_ARR: ["天蓬星", "天芮星", "天冲星", "天辅星", "天禽星", "天心星", "天柱星", "天任星", "天英星"],
  STAR: {
    /** 英文 */
    DUBHE: "天蓬星",
    MERAK: "天芮星",
    PHECDA: "天冲星",
    MEGREZ: "天辅星",
    ALIOTH: "天禽星",
    MIZAR: "天心星",
    ALKAID: "天柱星",
    IQ: "天任星",
    CVn5: "天英星",
    /** 中文-奇门 */
    TIAN_PENG: "天蓬星",
    TIAN_RUI: "天芮星",
    TIAN_CHONG: "天冲星",
    TIAN_FU: "天辅星",
    TIAN_QIN: "天禽星",
    TIAN_XIN: "天心星",
    TIAN_ZHU: "天柱星",
    TIAN_REN: "天任星",
    TIAN_YING: "天英星",
    /** 中文-北斗 */
    TIAN_SHU: "天蓬星",
    TIAN_XUAN: "天芮星",
    TIAN_JI: "天冲星",
    TIAN_QUAN: "天辅星",
    TIAN_HENG: "天禽星",
    KAI_YANG: "天心星",
    YAO_GUANG: "天柱星",
    DONG_MING: "天任星",
    YIN_YUAN: "天英星",
    /** 中文 */
    TAN_LANG: "天蓬星",
    JU_MENG: "天芮星",
    LU_CUN: "天冲星",
    WEN_QU: "天辅星",
    LIAN_ZHEN: "天禽星",
    WU_QU: "天心星",
    PO_JUN: "天柱星",
    ZUO_FU: "天任星",
    YOU_BI: "天英星",
    /** 简写-奇门 */
    TP: "天蓬星",
    TR: "天芮星",
    TC: "天冲星",
    TF: "天辅星",
    TQIN: "天禽星",
    TXIN: "天心星",
    TZ: "天柱星",
    TREN: "天任星",
    TY: "天英星",
    /** 简写-北斗 */
    TS: "天蓬星",
    TX: "天芮星",
    TJ: "天冲星",
    TQ: "天辅星",
    YH: "天禽星",
    KY: "天心星",
    YG: "天柱星",
    DM: "天任星",
    YY: "天英星",
    /** 简写 */
    TL: "天蓬星",
    JM: "天芮星",
    LC: "天冲星",
    WEQ: "天辅星",
    LZ: "天禽星",
    WUQ: "天心星",
    PJ: "天柱星",
    ZF: "天任星",
    YB: "天英星"
  }
};
