// tao_name/dist/sexagenaryCycle/sexagenaryCycle.js
const obj = {
  /**
   * 六十天干地支
   */
  SEXAGENARY_CYCLE_ARR: ["甲子", "乙丑", "丙寅", "丁卯", "戊辰", "己巳", "庚午", "辛未", "壬申", "癸酉", "甲戌", "乙亥", "丙子", "丁丑", "戊寅", "己卯", "庚辰", "辛巳", "壬午", "癸未", "甲申", "乙酉", "丙戌", "丁亥", "戊子", "己丑", "庚寅", "辛卯", "壬辰", "癸巳", "甲午", "乙未", "丙申", "丁酉", "戊戌", "己亥", "庚子", "辛丑", "壬寅", "癸卯", "甲辰", "乙巳", "丙午", "丁未", "戊申", "己酉", "庚戌", "辛亥", "壬子", "癸丑", "甲寅", "乙卯", "丙辰", "丁巳", "戊午", "己未", "庚申", "辛酉", "壬戌", "癸亥"],
  SEXAGENARY_CYCLE: {
    /** 英文写法 */
    MATH_Jan: "甲子",
    ETH_Feb: "乙丑",
    PROP_Mar: "丙寅",
    BUT_Apr: "丁卯",
    PENT_May: "戊辰",
    HEX_Jun: "己巳",
    HEPT_Jul: "庚午",
    OCT_Aug: "辛未",
    NON_Sep: "壬申",
    DEC_Oct: "癸酉",
    MATH_Nov: "甲戌",
    ETH_Dec: "乙亥",
    PROP_Jan: "丙子",
    BUT_Feb: "丁丑",
    PENT_Mar: "戊寅",
    HEX_Apr: "己卯",
    HEPT_May: "庚辰",
    OCT_Jun: "辛巳",
    NON_Jul: "壬午",
    DEC_Aug: "癸未",
    METH_Sep: "甲申",
    ETH_Oct: "乙酉",
    PROP_Nov: "丙戌",
    BUT_Dec: "丁亥",
    PENT_Jan: "戊子",
    HEX_Feb: "己丑",
    HEPT_Mar: "庚寅",
    OCT_Apr: "辛卯",
    NON_May: "壬辰",
    DEC_Jun: "癸巳",
    METH_Jul: "甲午",
    ETH_Aug: "乙未",
    PROP_Sep: "丙申",
    BUT_Oct: "丁酉",
    PENT_Nov: "戊戌",
    HEX_Dec: "己亥",
    HEPT_Jan: "庚子",
    OCT_Feb: "辛丑",
    NON_Mar: "壬寅",
    DEC_Apr: "癸卯",
    METH_May: "甲辰",
    ETH_Jun: "乙巳",
    PROP_Jul: "丙午",
    BUT_Aug: "丁未",
    PENT_Sep: "戊申",
    HEX_Oct: "己酉",
    HEPT_Nov: "庚戌",
    OCT_Dec: "辛亥",
    NON_Jan: "壬子",
    DEC_Feb: "癸丑",
    METH_Mar: "甲寅",
    ETH_Apr: "乙卯",
    PROP_May: "丙辰",
    BUT_Jun: "丁巳",
    PENT_Jul: "戊午",
    HEX_Aug: "己未",
    HEPT_Sep: "庚申",
    OCT_Oct: "辛酉",
    NON_Nov: "壬戌",
    DEC_Dec: "癸亥",
    /** 拼音写法 */
    JIA_zi: "甲子",
    YI_chou: "乙丑",
    BING_yin: "丙寅",
    DING_mao: "丁卯",
    WU_chen: "戊辰",
    JI_si: "己巳",
    GENG_wx: "庚午",
    XIN_wei: "辛未",
    REN_shen: "壬申",
    GUI_you: "癸酉",
    JIA_xu: "甲戌",
    YI_hai: "乙亥",
    BING_zi: "丙子",
    DING_chou: "丁丑",
    WU_yin: "戊寅",
    JI_mao: "己卯",
    GENG_chen: "庚辰",
    XIN_si: "辛巳",
    REN_wu: "壬午",
    GUI_wei: "癸未",
    JIA_shen: "甲申",
    YI_you: "乙酉",
    BING_xu: "丙戌",
    DING_hai: "丁亥",
    WU_zi: "戊子",
    JI_chou: "己丑",
    GENG_yin: "庚寅",
    XIN_mao: "辛卯",
    REN_chen: "壬辰",
    GUI_si: "癸巳",
    JIA_wu: "甲午",
    YI_wei: "乙未",
    BING_shen: "丙申",
    DING_you: "丁酉",
    WU_xu: "戊戌",
    JI_hai: "己亥",
    GENG_zi: "庚子",
    XIN_chou: "辛丑",
    REN_yin: "壬寅",
    GUI_mao: "癸卯",
    JIA_chen: "甲辰",
    YI_si: "乙巳",
    BING_wu: "丙午",
    DING_wei: "丁未",
    WU_shen: "戊申",
    JI_you: "己酉",
    GENG_xu: "庚戌",
    XIN_hai: "辛亥",
    REN_zi: "壬子",
    GUI_chou: "癸丑",
    JIA_yin: "甲寅",
    YI_mao: "乙卯",
    BING_chen: "丙辰",
    DING_si: "丁巳",
    WU_wu: "戊午",
    JI_wei: "己未",
    GENG_shen: "庚申",
    XIN_you: "辛酉",
    REN_xu: "壬戌",
    GUI_hai: "癸亥"
  }
};
obj.SC_ARR = obj.SEXAGENARY_CYCLE_ARR;
obj.SC = obj.SEXAGENARY_CYCLE;
module.exports = obj;
