// liuren-ts-lib/dist/esm/index.js
// src/maps/sixtyJiaZi.ts
var sixtyJiaZi = [
  "\u7532\u5B50",
  "\u4E59\u4E11",
  "\u4E19\u5BC5",
  "\u4E01\u536F",
  "\u620A\u8FB0",
  "\u5DF1\u5DF3",
  "\u5E9A\u5348",
  "\u8F9B\u672A",
  "\u58EC\u7533",
  "\u7678\u9149",
  "\u7532\u620C",
  "\u4E59\u4EA5",
  "\u4E19\u5B50",
  "\u4E01\u4E11",
  "\u620A\u5BC5",
  "\u5DF1\u536F",
  "\u5E9A\u8FB0",
  "\u8F9B\u5DF3",
  "\u58EC\u5348",
  "\u7678\u672A",
  "\u7532\u7533",
  "\u4E59\u9149",
  "\u4E19\u620C",
  "\u4E01\u4EA5",
  "\u620A\u5B50",
  "\u5DF1\u4E11",
  "\u5E9A\u5BC5",
  "\u8F9B\u536F",
  "\u58EC\u8FB0",
  "\u7678\u5DF3",
  "\u7532\u5348",
  "\u4E59\u672A",
  "\u4E19\u7533",
  "\u4E01\u9149",
  "\u620A\u620C",
  "\u5DF1\u4EA5",
  "\u5E9A\u5B50",
  "\u8F9B\u4E11",
  "\u58EC\u5BC5",
  "\u7678\u536F",
  "\u7532\u8FB0",
  "\u4E59\u5DF3",
  "\u4E19\u5348",
  "\u4E01\u672A",
  "\u620A\u7533",
  "\u5DF1\u9149",
  "\u5E9A\u620C",
  "\u8F9B\u4EA5",
  "\u58EC\u5B50",
  "\u7678\u4E11",
  "\u7532\u5BC5",
  "\u4E59\u536F",
  "\u4E19\u8FB0",
  "\u4E01\u5DF3",
  "\u620A\u5348",
  "\u5DF1\u672A",
  "\u5E9A\u7533",
  "\u8F9B\u9149",
  "\u58EC\u620C",
  "\u7678\u4EA5"
];
var sixtyJiaZi_default = sixtyJiaZi;

// src/common/date.ts
import { EightChar, LunarHour, SixtyCycle, SolarDay } from "tyme4ts";

// src/maps/ma.ts
var YiMa = {
  "\u5BC5": "\u7533",
  "\u5348": "\u7533",
  "\u620C": "\u7533",
  "\u7533": "\u5BC5",
  "\u5B50": "\u5BC5",
  "\u8FB0": "\u5BC5",
  "\u5DF3": "\u4EA5",
  "\u9149": "\u4EA5",
  "\u4E11": "\u4EA5",
  "\u4EA5": "\u5DF3",
  "\u536F": "\u5DF3",
  "\u672A": "\u5DF3"
};
var DingMa = {
  "\u7532\u5B50": "\u536F",
  "\u7532\u620C": "\u4E11",
  "\u7532\u7533": "\u4EA5",
  "\u7532\u5348": "\u9149",
  "\u7532\u8FB0": "\u672A",
  "\u7532\u5BC5": "\u5DF3"
};
var TianMa = {
  "\u672A": "\u8FB0",
  "\u4E11": "\u8FB0",
  "\u5348": "\u5BC5",
  "\u5B50": "\u5BC5",
  "\u5DF3": "\u5B50",
  "\u4EA5": "\u5B50",
  "\u8FB0": "\u620C",
  "\u620C": "\u620C",
  "\u536F": "\u7533",
  "\u9149": "\u7533",
  "\u7533": "\u5348",
  "\u5BC5": "\u5348"
};

// src/maps/yueJiang.ts
var YueJiangName = {
  "\u4EA5": "\u767B\u660E",
  "\u620C": "\u6CB3\u9B41",
  "\u9149": "\u4ECE\u9B41",
  "\u7533": "\u4F20\u9001",
  "\u672A": "\u5C0F\u5409",
  "\u5348": "\u80DC\u5149",
  "\u5DF3": "\u592A\u4E59",
  "\u8FB0": "\u5929\u7F61",
  "\u536F": "\u592A\u51B2",
  "\u5BC5": "\u529F\u66F9",
  "\u4E11": "\u5927\u5409",
  "\u5B50": "\u795E\u540E"
};
var YueJiang = {
  "\u96E8\u6C34": "\u4EA5",
  "\u60CA\u86F0": "\u4EA5",
  "\u6625\u5206": "\u620C",
  "\u6E05\u660E": "\u620C",
  "\u8C37\u96E8": "\u9149",
  "\u7ACB\u590F": "\u9149",
  "\u5C0F\u6EE1": "\u7533",
  "\u8292\u79CD": "\u7533",
  "\u590F\u81F3": "\u672A",
  "\u5C0F\u6691": "\u672A",
  "\u5927\u6691": "\u5348",
  "\u7ACB\u79CB": "\u5348",
  "\u5904\u6691": "\u5DF3",
  "\u767D\u9732": "\u5DF3",
  "\u79CB\u5206": "\u8FB0",
  "\u5BD2\u9732": "\u8FB0",
  "\u971C\u964D": "\u536F",
  "\u7ACB\u51AC": "\u536F",
  "\u5C0F\u96EA": "\u5BC5",
  "\u5927\u96EA": "\u5BC5",
  "\u51AC\u81F3": "\u4E11",
  "\u5C0F\u5BD2": "\u4E11",
  "\u5927\u5BD2": "\u5B50",
  "\u7ACB\u6625": "\u5B50"
};

// src/common/date.ts
var getDateBySiZhu = (year, month, day, hour) => {
  const eightChar = new EightChar(year, month, day, hour);
  const solar = eightChar.getSolarTimes(2e3, 2050)[0];
  return getDateByObj(/* @__PURE__ */ new Date(`${solar.getYear()}-${solar.getMonth()}-${solar.getDay()} ${solar.getHour()}:00`));
};
var getDateByObj = (date) => {
  const result = {
    bazi: "",
    date: "",
    kong: [],
    yima: "",
    yuejiang: "",
    xun: "",
    dingma: "",
    tianma: ""
  };
  const year = date.getFullYear();
  const month = date.getMonth() + 1;
  const day = date.getDate();
  const hour = date.getHours();
  const minute = date.getMinutes();
  const second = date.getSeconds();
  const solar = SolarDay.fromYmd(year, month, day);
  const lunar = solar.getLunarDay();
  const eightChar = LunarHour.fromYmdHms(lunar.getYear(), lunar.getMonth(), lunar.getDay(), hour, minute, second).getEightChar();
  result.bazi = `${eightChar.getYear()} ${eightChar.getMonth()} ${eightChar.getDay()} ${eightChar.getHour()}`;
  result.date = `${year}\u5E74${month}\u6708${day}\u65E5 ${hour}\u65F6${minute}\u5206`;
  const sixtyCycle = SixtyCycle.fromName(eightChar.getDay().toString());
  result.kong = sixtyCycle.getExtraEarthBranches().map((item) => item.toString());
  result.xun = sixtyCycle.getTen().toString();
  const yueLing = eightChar.getMonth().toString().substring(1, 2);
  const jieQi = solar.getTerm().toString();
  result.yuejiang = YueJiang[jieQi];
  result.tianma = TianMa[yueLing];
  result.dingma = DingMa[result.xun];
  const hourBranch = eightChar.getDay().toString().substring(1, 2);
  if (YiMa.hasOwnProperty(hourBranch)) {
    result.yima = YiMa[hourBranch];
  } else {
    result.yima = "";
  }
  return result;
};

// src/maps/ganZhi.ts
var DiZhiArray = ["\u5B50", "\u4E11", "\u5BC5", "\u536F", "\u8FB0", "\u5DF3", "\u5348", "\u672A", "\u7533", "\u9149", "\u620C", "\u4EA5"];
var LiuHe = {
  "\u5B50": "\u4E11",
  "\u4E11": "\u5B50",
  "\u5BC5": "\u4EA5",
  "\u536F": "\u620C",
  "\u8FB0": "\u9149",
  "\u5DF3": "\u7533",
  "\u5348": "\u672A",
  "\u672A": "\u5348",
  "\u7533": "\u5DF3",
  "\u9149": "\u8FB0",
  "\u620C": "\u536F",
  "\u4EA5": "\u5BC5"
};
var LiuChong = {
  "\u5B50": "\u5348",
  "\u4E11": "\u672A",
  "\u5BC5": "\u7533",
  "\u536F": "\u9149",
  "\u8FB0": "\u620C",
  "\u5DF3": "\u4EA5",
  "\u5348": "\u5B50",
  "\u672A": "\u4E11",
  "\u7533": "\u5BC5",
  "\u9149": "\u536F",
  "\u620C": "\u8FB0",
  "\u4EA5": "\u5DF3"
};
var DiZhiNumber = {
  "\u5B50": 1,
  "\u4E11": 2,
  "\u5BC5": 3,
  "\u536F": 4,
  "\u8FB0": 5,
  "\u5DF3": 6,
  "\u5348": 7,
  "\u672A": 8,
  "\u7533": 9,
  "\u9149": 10,
  "\u620C": 11,
  "\u4EA5": 12
};
var TianGanArray = ["\u7532", "\u4E59", "\u4E19", "\u4E01", "\u620A", "\u5DF1", "\u5E9A", "\u8F9B", "\u58EC", "\u7678"];
var TianGanIndex = {
  "\u7532": 0 + 1,
  "\u4E59": 1 + 1,
  "\u4E19": 2 + 1,
  "\u4E01": 3 + 1,
  "\u620A": 4 + 1,
  "\u5DF1": 5 + 1,
  "\u5E9A": 6 + 1,
  "\u8F9B": 7 + 1,
  "\u58EC": 8 + 1,
  "\u7678": 9 + 1
};
var TianGanWuHe = {
  "\u7532": "\u5DF1",
  "\u4E59": "\u5E9A",
  "\u4E19": "\u8F9B",
  "\u4E01": "\u58EC",
  "\u620A": "\u7678",
  "\u5DF1": "\u7532",
  "\u5E9A": "\u4E59",
  "\u8F9B": "\u4E19",
  "\u58EC": "\u4E01",
  "\u7678": "\u620A"
};

// src/liuren/dunGan.ts
var WuZiYuanDun = {
  "\u7532": 0,
  "\u5DF1": 0,
  "\u4E59": 2,
  "\u5E9A": 2,
  "\u4E19": 4,
  "\u8F9B": 4,
  "\u4E01": 6,
  "\u58EC": 6,
  "\u620A": 8,
  "\u7678": 8
};
var getYuanDun = (gan) => {
  const result = {
    "\u5B50": "",
    "\u4E11": "",
    "\u5BC5": "",
    "\u536F": "",
    "\u8FB0": "",
    "\u5DF3": "",
    "\u5348": "",
    "\u672A": "",
    "\u7533": "",
    "\u9149": "",
    "\u620C": "",
    "\u4EA5": ""
  };
  const startIndex = WuZiYuanDun[gan] ?? 0;
  for (let i = 0; i < 12; i++) {
    const zhi = DiZhiArray[i];
    const tianGan = TianGanArray[(startIndex + i) % 10];
    result[zhi] = tianGan;
  }
  return result;
};
var getDunGan = (date, tiandipan) => {
  const dunGan = {
    "\u5B50": "",
    "\u4E11": "",
    "\u5BC5": "",
    "\u536F": "",
    "\u8FB0": "",
    "\u5DF3": "",
    "\u5348": "",
    "\u672A": "",
    "\u7533": "",
    "\u9149": "",
    "\u620C": "",
    "\u4EA5": ""
  };
  const xun = date.xun;
  const kong = date.kong;
  const gan = xun.substring(0, 1);
  const zhi = xun.substring(1, 2);
  const tianPan = tiandipan.\u5929\u76D8;
  const zhiIndex = DiZhiArray.indexOf(zhi);
  for (let i = 0; i < 10; i++) {
    let index = zhiIndex + i;
    const gan2 = TianGanArray[i];
    if (index > 11) index = index - 12;
    const gong = DiZhiArray[index];
    dunGan[gong] = gan2;
  }
  return dunGan;
};
var getChuJian = (date) => {
  const riGan = date.bazi.split(" ")[2].substring(0, 1);
  return getYuanDun(riGan);
};
var getFuJian = (date) => {
  const shiGan = date.bazi.split(" ")[3].substring(0, 1);
  return getYuanDun(shiGan);
};

// src/liuren/jianChu.ts
var JianChuArray = ["\u5EFA", "\u9664", "\u6EE1", "\u5E73", "\u5B9A", "\u6267", "\u7834", "\u5371", "\u6210", "\u6536", "\u5F00", "\u95ED"];
var getJianChu = (date, tiandipan) => {
  const result = {
    "\u5B50": "",
    "\u4E11": "",
    "\u5BC5": "",
    "\u536F": "",
    "\u8FB0": "",
    "\u5DF3": "",
    "\u5348": "",
    "\u672A": "",
    "\u7533": "",
    "\u9149": "",
    "\u620C": "",
    "\u4EA5": ""
  };
  const yueZhi = date.bazi.split(" ")[1].substring(1, 2);
  const tianPan = tiandipan.\u5929\u76D8;
  let startIndex = 0;
  for (let i = 0; i < 12; i++) {
    if (tianPan[i] === yueZhi) {
      startIndex = i;
      break;
    }
  }
  for (let i = 0; i < 12; i++) {
    let index = startIndex + i;
    if (index > 11) index -= 12;
    const gong = DiZhiArray[index];
    result[gong] = JianChuArray[i];
  }
  return result;
};

// src/maps/jiGong.ts
var JiGong = {
  "\u7532": "\u5BC5",
  "\u4E59": "\u8FB0",
  "\u4E19": "\u5DF3",
  "\u620A": "\u5DF3",
  "\u4E01": "\u672A",
  "\u5DF1": "\u672A",
  "\u5E9A": "\u7533",
  "\u8F9B": "\u620C",
  "\u58EC": "\u4EA5",
  "\u7678": "\u4E11"
};

// src/maps/wuXing.ts
var WuXing = {
  "\u5B50": "\u6C34",
  "\u4E11": "\u571F",
  "\u5BC5": "\u6728",
  "\u536F": "\u6728",
  "\u8FB0": "\u571F",
  "\u5DF3": "\u706B",
  "\u5348": "\u706B",
  "\u672A": "\u571F",
  "\u7533": "\u91D1",
  "\u9149": "\u91D1",
  "\u620C": "\u571F",
  "\u4EA5": "\u6C34",
  "\u7532": "\u6728",
  "\u4E59": "\u6728",
  "\u4E19": "\u706B",
  "\u4E01": "\u706B",
  "\u620A": "\u571F",
  "\u5DF1": "\u571F",
  "\u5E9A": "\u91D1",
  "\u8F9B": "\u91D1",
  "\u58EC": "\u6C34",
  "\u7678": "\u6C34"
};
var Relation = {
  //水
  "\u6C34\u6728": "\u751F",
  "\u6C34\u706B": "\u514B",
  "\u6C34\u571F": "\u8D3C",
  "\u6C34\u91D1": "\u517B",
  "\u6C34\u6C34": "\u6BD4",
  //木
  "\u6728\u6C34": "\u517B",
  "\u6728\u706B": "\u751F",
  "\u6728\u571F": "\u514B",
  "\u6728\u91D1": "\u8D3C",
  "\u6728\u6728": "\u6BD4",
  //火
  "\u706B\u6C34": "\u8D3C",
  "\u706B\u706B": "\u6BD4",
  "\u706B\u571F": "\u751F",
  "\u706B\u91D1": "\u514B",
  "\u706B\u6728": "\u517B",
  //土
  "\u571F\u6C34": "\u514B",
  "\u571F\u706B": "\u517B",
  "\u571F\u571F": "\u6BD4",
  "\u571F\u91D1": "\u751F",
  "\u571F\u6728": "\u8D3C",
  //金
  "\u91D1\u6C34": "\u751F",
  "\u91D1\u706B": "\u8D3C",
  "\u91D1\u571F": "\u517B",
  "\u91D1\u91D1": "\u6BD4",
  "\u91D1\u6728": "\u514B"
};
var LiuQin = {
  "\u751F": "\u5B50\u5B59",
  "\u514B": "\u59BB\u8D22",
  "\u8D3C": "\u5B98\u9B3C",
  "\u517B": "\u7236\u6BCD",
  "\u6BD4": "\u5144\u5F1F"
};

// src/liuren/siKe.ts
var getSiKe = (date, tianDiPan) => {
  const siKe = {
    "\u4E00\u8BFE": [],
    "\u4E8C\u8BFE": [],
    "\u4E09\u8BFE": [],
    "\u56DB\u8BFE": []
  };
  const riChen = date.bazi.split(" ")[2];
  const riGan = riChen.substring(0, 1);
  const riZhi = riChen.substring(1, 2);
  const jiGong = JiGong[riGan];
  let shangShen = getShangShen(tianDiPan, jiGong);
  siKe.\u4E00\u8BFE = [`${shangShen}${riGan}`, getTianJiang(tianDiPan, shangShen)];
  let oldShangShen = shangShen;
  shangShen = getShangShen(tianDiPan, shangShen);
  siKe.\u4E8C\u8BFE = [`${shangShen}${oldShangShen}`, getTianJiang(tianDiPan, shangShen)];
  oldShangShen = shangShen;
  shangShen = getShangShen(tianDiPan, riZhi);
  siKe.\u4E09\u8BFE = [`${shangShen}${riZhi}`, getTianJiang(tianDiPan, shangShen)];
  oldShangShen = shangShen;
  shangShen = getShangShen(tianDiPan, shangShen);
  siKe.\u56DB\u8BFE = [`${shangShen}${oldShangShen}`, getTianJiang(tianDiPan, shangShen)];
  return siKe;
};
var getXiaShen = (tianDiPan, zhi) => {
  let TianPan = tianDiPan["\u5929\u76D8"];
  let DiPan = tianDiPan["\u5730\u76D8"];
  for (let i = 0; i < 12; i++) {
    if (TianPan[i] === zhi) {
      return DiPan[i];
    }
  }
  return "";
};
var getShangShen = (tianDiPan, zhi) => {
  let DiPan = tianDiPan["\u5730\u76D8"];
  let TianPan = tianDiPan["\u5929\u76D8"];
  for (let i = 0; i < 12; i++) {
    if (DiPan[i] === zhi) {
      return TianPan[i];
    }
  }
  return "";
};
var getTianJiang = (tianDiPan, zhi) => {
  let TianJiang = tianDiPan["\u5929\u5C06"];
  let TianPan = tianDiPan["\u5929\u76D8"];
  for (let i = 0; i < 12; i++) {
    if (TianPan[i] === zhi) {
      return TianJiang[i];
    }
  }
  return "";
};

// src/sanchuan.json
var sanchuan_default = {
  \u620A\u8FB0: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u5348",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5BC5\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u8FB0\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5DF3\u4EA5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u620A\u5BC5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5348\u9149",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u620A\u5B50: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u4E11",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5348\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u620A\u620C: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u620C\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5BC5\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4E11\u5348",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5DF3\u4EA5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u620A\u7533: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u5348",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u620A\u5348: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u5348",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u5DF1\u5DF3: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4EA5\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u7533\u5348",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u4E11\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u5DF1\u536F: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u9149\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5B50\u4EA5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5B50\u5348",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u4E11\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u5DF1\u4E11: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u4E11",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u8FB0\u620C",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5DF3\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u620C\u8FB0",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u5DF1\u4EA5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4E11\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u4E11",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u5DF1\u9149: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5B50\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4E11\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u7533",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u672A\u4E11",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5348\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u5DF1\u672A: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u4E11",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4EA5\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u8FB0\u8FB0",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5DF3\u5DF3",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5348\u5348",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4E11\u620C",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u7533\u7533",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u9149\u9149",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u620C\u620C",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E19\u5BC5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u7533\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E19\u5B50: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u8FB0\u5B50",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u77E5\u4E00"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E19\u620C: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5DF3\u4E11",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u7533\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4E11\u5348",
      \u683C\u5C40: "\u77E5\u4E00"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E19\u7533: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u620C\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E19\u5348: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u9149\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u9149\u620C",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E19\u8FB0: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u7533\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u5348",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    }
  ],
  \u4E01\u536F: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u9149\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5B50\u4EA5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5B50\u5348",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E01\u4E11: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u4E11",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u8FB0\u620C",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u9149\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u620C\u8FB0",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E01\u4EA5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u672A\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u4E11",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u9149\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u620C\u5BC5",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E01\u9149: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5B50\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5DF3\u5DF3",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u672A\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u672A\u4E11",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u536F\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u5143\u9996"
    }
  ],
  \u4E01\u672A: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u4E11",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4EA5\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u8FB0\u8FB0",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5DF3\u5DF3",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5348\u5348",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4E11\u620C",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u9149\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u620C\u620C",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E01\u5DF3: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u7533\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u9149\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4EA5\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u5E9A\u5348: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5DF3\u8FB0",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u672A\u9149",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u5E9A\u8FB0: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u672A\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u5E9A\u5BC5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u5E9A\u5B50: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u9149\u5B50",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u5E9A\u620C: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4E11\u5348",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5DF3\u8FB0",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u5E9A\u7533: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4E11\u4E11",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u672A\u672A",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u9149",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u4EA5",
      \u683C\u5C40: "\u516B\u4E13"
    }
  ],
  \u8F9B\u672A: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u8FB0\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u4E11\u4E11",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u8FB0",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4EA5\u672A",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u672A",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u8FB0\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4E11\u620C",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u7533",
      \u683C\u5C40: "\u6634\u661F"
    }
  ],
  \u8F9B\u5DF3: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u8FB0\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5BC5\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5BC5\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u4EA5\u7533",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u672A\u7533",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u8F9B\u536F: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u672A\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5B50",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5B50\u4EA5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5B50\u5348",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u8F9B\u4E11: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5DF3\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u4E11",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u672A\u8FB0",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u672A\u672A",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    }
  ],
  \u8F9B\u4EA5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u7533\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    }
  ],
  \u8F9B\u9149: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5348\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5B50\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u9149\u9149",
      \u683C\u5C40: "\u522B\u8D23"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u7532\u5B50: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u9149\u8FB0",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    }
  ],
  \u7532\u620C: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u672A\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5DF3\u5BC5",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u7532\u7533: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u7533\u5B50",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5DF3\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    }
  ],
  \u7532\u5348: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5348\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5DF3\u5BC5",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u7532\u8FB0: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5B50\u8FB0",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5DF3\u5BC5",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u7532\u5BC5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5348\u5348",
      \u683C\u5C40: "\u516B\u4E13"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u8FB0\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u4EA5",
      \u683C\u5C40: "\u516B\u4E13"
    }
  ],
  \u4E59\u4E11: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u4E11\u620C",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u620C\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u8FB0\u620C",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    }
  ],
  \u4E59\u4EA5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u672A\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u4EA5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u620C\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E59\u9149: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5DF3\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u672A\u5348",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u536F",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u620C\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5B50\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5B50\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u4E11",
      \u683C\u5C40: "\u6BD4\u7528"
    }
  ],
  \u4E59\u672A: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u4EA5\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5BC5\u5DF3",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u536F\u5348",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u4E11",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u620C\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u620C\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u536F\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u8FB0\u620C",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E59\u5DF3: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5DF3\u4E11",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u7533",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u7533\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u620C\u4E11",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u4E59\u536F: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5B50\u4EA5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u536F\u5B50",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u58EC\u7533: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u7533\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u7533\u5BC5",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u58EC\u5348: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5B50\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5348\u5B50",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u58EC\u8FB0: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u620C\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u4E11\u8FB0",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u672A\u5B50",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u7533\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u5B50\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u8FB0\u620C",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u58EC\u5BC5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5DF3\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u7533\u5BC5",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u5348\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5BC5\u5DF3",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u58EC\u5B50: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5348\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u9149\u5B50",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u5B50\u5348",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u7533\u5348",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u536F",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u58EC\u620C: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u5BC5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u620C",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4EA5\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u9149\u5BC5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4E11\u7533",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u8FB0\u5BC5",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    }
  ],
  \u7678\u9149: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5348\u5DF3",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u5B50\u4E11",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5B50\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u536F\u5B50",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5DF3\u536F",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u7678\u672A: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u8FB0\u536F",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u5BC5\u7533",
      \u683C\u5C40: "\u6634\u661F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u672A\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u620C\u536F",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4E11\u672A",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u672A\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u536F\u4E11",
      \u683C\u5C40: "\u9065\u514B"
    }
  ],
  \u7678\u5DF3: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5BC5\u4E11",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u7533\u9149",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u9149\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u7533\u4EA5\u5BC5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4EA5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u672A\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u7678\u536F: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5B50\u4EA5",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u5DF3\u5348",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u9149\u4EA5",
      \u683C\u5C40: "\u9065\u514B"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u5B50\u536F",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4EA5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u9149\u536F",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u672A\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u4EA5\u9149",
      \u683C\u5C40: "\u6D89\u5BB3"
    }
  ],
  \u7678\u4E11: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u5B50\u4EA5\u620C",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5BC5\u536F\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u5DF3\u672A",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4EA5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u4E11\u672A",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4E11\u9149",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u672A\u8FB0",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4EA5\u9149\u672A",
      \u683C\u5C40: "\u91CD\u5BA1"
    }
  ],
  \u7678\u4EA5: [
    {
      \u5E72\u652F\u7EC4\u5408: "\u620C\u9149\u7533",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u620C\u672A",
      \u683C\u5C40: "\u4F0F\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u5BC5\u536F",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u4E11\u536F\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u8FB0\u672A\u620C",
      \u683C\u5C40: "\u5143\u9996"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u9149\u4E11\u5DF3",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5348\u4EA5\u8FB0",
      \u683C\u5C40: "\u91CD\u5BA1"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u4EA5\u5DF3",
      \u683C\u5C40: "\u53CD\u541F"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u536F\u620C\u5DF3",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u536F\u4EA5",
      \u683C\u5C40: "\u6D89\u5BB3"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u5DF3\u5BC5\u4EA5",
      \u683C\u5C40: "\u6BD4\u7528"
    },
    {
      \u5E72\u652F\u7EC4\u5408: "\u672A\u5DF3\u536F",
      \u683C\u5C40: "\u9065\u514B"
    }
  ]
};

// src/liuren/sanChuan.ts
var getSanChuan = (siKe, tiandipan) => {
  let sanChuan = {
    "\u521D\u4F20": [],
    "\u4E2D\u4F20": [],
    "\u672B\u4F20": [],
    "\u8BFE\u4F53": ""
  };
  const ke1_shang = siKe.\u4E00\u8BFE[0][0];
  const ke1_xia = siKe.\u4E00\u8BFE[0][1];
  const riGan = ke1_xia;
  const riJi = JiGong[ke1_xia];
  const ke2_shang = siKe.\u4E8C\u8BFE[0][0];
  const ke2_xia = siKe.\u4E8C\u8BFE[0][1];
  const ke3_shang = siKe.\u4E09\u8BFE[0][0];
  const ke3_xia = siKe.\u4E09\u8BFE[0][1];
  const ke4_shang = siKe.\u56DB\u8BFE[0][0];
  const ke4_xia = siKe.\u56DB\u8BFE[0][1];
  const siKeArray = [];
  siKeArray.push(ke1_shang + ke1_xia);
  siKeArray.push(ke2_shang + ke2_xia);
  siKeArray.push(ke3_shang + ke3_xia);
  siKeArray.push(ke4_shang + ke4_xia);
  const day = riGan + ke3_xia;
  const index = DiZhiArray.indexOf(ke1_shang);
  const ganZhi = sanchuan_default[day][index].\u5E72\u652F\u7EC4\u5408;
  sanChuan.\u8BFE\u4F53 = sanchuan_default[day][index].\u683C\u5C40;
  sanChuan.\u521D\u4F20 = [ganZhi.substring(0, 1), "", "", ""];
  sanChuan.\u4E2D\u4F20 = [ganZhi.substring(1, 2), "", "", ""];
  sanChuan.\u672B\u4F20 = [ganZhi.substring(2, 3), "", "", ""];
  return sanChuan;
};
var fillSanChuan = (sanChuan, tiandipan, dunGan, riGan) => {
  const chu = sanChuan.\u521D\u4F20[0];
  let tianJiang = getTianJiang(tiandipan, chu);
  let relation = getLiuQin(riGan, chu);
  let dun = dunGan[chu];
  sanChuan.\u521D\u4F20 = [chu, tianJiang, relation, dun];
  const zhong = sanChuan.\u4E2D\u4F20[0];
  tianJiang = getTianJiang(tiandipan, zhong);
  relation = getLiuQin(riGan, zhong);
  dun = dunGan[zhong];
  sanChuan.\u4E2D\u4F20 = [zhong, tianJiang, relation, dun];
  const mo = sanChuan.\u672B\u4F20[0];
  tianJiang = getTianJiang(tiandipan, mo);
  relation = getLiuQin(riGan, mo);
  dun = dunGan[mo];
  sanChuan.\u672B\u4F20 = [mo, tianJiang, relation, dun];
  return sanChuan;
};
var getGanZhi2WuXing = (ganZhi) => {
  let result = "";
  for (let i = 0; i < ganZhi.length; i++) {
    result += WuXing[ganZhi[i]];
  }
  return result;
};
var getGanZhi2Relation = (ganZhi) => {
  const wuxing = getGanZhi2WuXing(ganZhi);
  const relation = Relation[wuxing];
  return relation;
};
var getLiuQin = (item1, item2) => {
  const wuxing1 = getGanZhi2WuXing(item1);
  const wuxing2 = getGanZhi2WuXing(item2);
  const relation = Relation[wuxing1 + wuxing2];
  return LiuQin[relation];
};

// src/maps/shensha.ts
var riDe = {
  "\u7532": "\u5BC5",
  "\u4E59": "\u7533",
  "\u4E19": "\u5DF3",
  "\u4E01": "\u4EA5",
  "\u620A": "\u5DF3",
  "\u5DF1": "\u5BC5",
  "\u5E9A": "\u7533",
  "\u8F9B": "\u5DF3",
  "\u58EC": "\u4EA5",
  "\u7678": "\u5DF3"
};
var riLu = {
  "\u7532": "\u5BC5",
  "\u4E59": "\u536F",
  "\u4E19": "\u5DF3",
  "\u4E01": "\u5348",
  "\u620A": "\u5DF3",
  "\u5DF1": "\u5348",
  "\u5E9A": "\u7533",
  "\u8F9B": "\u9149",
  "\u58EC": "\u4EA5",
  "\u7678": "\u5B50"
};
var ShengQi = {
  "\u5BC5": "\u5B50",
  "\u536F": "\u4E11",
  "\u8FB0": "\u5BC5",
  "\u5DF3": "\u536F",
  "\u5348": "\u8FB0",
  "\u672A": "\u5DF3",
  "\u7533": "\u5348",
  "\u9149": "\u672A",
  "\u620C": "\u7533",
  "\u4EA5": "\u9149",
  "\u5B50": "\u620C",
  "\u4E11": "\u4EA5"
};
var PoSui = {
  "\u5BC5": "\u9149",
  "\u7533": "\u9149",
  "\u5DF3": "\u9149",
  "\u4EA5": "\u9149",
  "\u5B50": "\u5DF3",
  "\u5348": "\u5DF3",
  "\u536F": "\u5DF3",
  "\u9149": "\u5DF3",
  "\u8FB0": "\u4E11",
  "\u620C": "\u4E11",
  "\u4E11": "\u4E11",
  "\u672A": "\u4E11"
};
var TianDe = {
  "\u5BC5": "\u4E01",
  "\u536F": "\u7533",
  "\u8FB0": "\u58EC",
  "\u5DF3": "\u8F9B",
  "\u5348": "\u4EA5",
  "\u672A": "\u7532",
  "\u7533": "\u7678",
  "\u9149": "\u5BC5",
  "\u620C": "\u4E19",
  "\u4EA5": "\u4E59",
  "\u5B50": "\u5DF3",
  "\u4E11": "\u5E9A"
};
var YueDe = {
  "\u5BC5": "\u4E19",
  "\u5348": "\u4E19",
  "\u620C": "\u4E19",
  "\u4EA5": "\u7532",
  "\u536F": "\u7532",
  "\u672A": "\u7532",
  "\u7533": "\u58EC",
  "\u5B50": "\u58EC",
  "\u8FB0": "\u58EC",
  "\u5DF3": "\u5E9A",
  "\u9149": "\u5E9A",
  "\u4E11": "\u5E9A"
};
var TianMa2 = {
  "\u5BC5": "\u5348",
  "\u7533": "\u5348",
  "\u536F": "\u7533",
  "\u9149": "\u7533",
  "\u8FB0": "\u620C",
  "\u620C": "\u620C",
  "\u5DF3": "\u5B50",
  "\u4EA5": "\u5B50",
  "\u5348": "\u5BC5",
  "\u5B50": "\u5BC5",
  "\u672A": "\u8FB0",
  "\u4E11": "\u8FB0"
};
var TianXi = {
  "\u5BC5": "\u620C",
  "\u536F": "\u4EA5",
  "\u8FB0": "\u5B50",
  "\u5DF3": "\u4E11",
  "\u5348": "\u5BC5",
  "\u672A": "\u536F",
  "\u7533": "\u8FB0",
  "\u9149": "\u5DF3",
  "\u620C": "\u5348",
  "\u4EA5": "\u672A",
  "\u5B50": "\u7533",
  "\u4E11": "\u9149"
};
var TianYi = {
  "\u5BC5": "\u620C",
  "\u536F": "\u4EA5",
  "\u8FB0": "\u5B50",
  "\u5DF3": "\u4E11",
  "\u5348": "\u5BC5",
  "\u672A": "\u536F",
  "\u7533": "\u8FB0",
  "\u9149": "\u5DF3",
  "\u620C": "\u5348",
  "\u4EA5": "\u672A",
  "\u5B50": "\u7533",
  "\u4E11": "\u9149"
};
var JieSha = {
  "\u7533": "\u5DF3",
  "\u5B50": "\u5DF3",
  "\u8FB0": "\u5DF3",
  "\u5DF3": "\u5BC5",
  "\u9149": "\u5BC5",
  "\u4E11": "\u5BC5",
  "\u5BC5": "\u4EA5",
  "\u5348": "\u4EA5",
  "\u620C": "\u4EA5",
  "\u4EA5": "\u7533",
  "\u536F": "\u7533",
  "\u672A": "\u7533"
};
var SangChe = {
  "\u5BC5": "\u9149",
  "\u536F": "\u9149",
  "\u8FB0": "\u9149",
  "\u5DF3": "\u5B50",
  "\u5348": "\u5B50",
  "\u672A": "\u5B50",
  "\u7533": "\u536F",
  "\u9149": "\u536F",
  "\u620C": "\u536F",
  "\u4EA5": "\u5348",
  "\u5B50": "\u5348",
  "\u4E11": "\u5348"
};
var FeiLian = {
  "\u5BC5": "\u620C",
  "\u536F": "\u5DF3",
  "\u8FB0": "\u5348",
  "\u5DF3": "\u672A",
  "\u5348": "\u7533",
  "\u672A": "\u9149",
  "\u7533": "\u620C",
  "\u9149": "\u4EA5",
  "\u620C": "\u5B50",
  "\u4EA5": "\u4E11",
  "\u5B50": "\u5BC5",
  "\u4E11": "\u536F"
};
var SiQiu = {
  "\u5BC5": "\u4E11",
  "\u536F": "\u4E11",
  "\u8FB0": "\u4E11",
  "\u5DF3": "\u8FB0",
  "\u5348": "\u8FB0",
  "\u672A": "\u8FB0",
  "\u7533": "\u672A",
  "\u9149": "\u672A",
  "\u620C": "\u672A",
  "\u4EA5": "\u620C",
  "\u5B50": "\u620C",
  "\u4E11": "\u620C"
};
var SiMu = {
  "\u5BC5": "\u672A",
  "\u536F": "\u672A",
  "\u8FB0": "\u672A",
  "\u5DF3": "\u620C",
  "\u5348": "\u620C",
  "\u672A": "\u620C",
  "\u7533": "\u4E11",
  "\u9149": "\u4E11",
  "\u620C": "\u4E11",
  "\u4EA5": "\u8FB0",
  "\u5B50": "\u8FB0",
  "\u4E11": "\u8FB0"
};
var TianGui = {
  "\u5BC5": "\u9149",
  "\u536F": "\u9149",
  "\u8FB0": "\u9149",
  "\u5DF3": "\u5B50",
  "\u5348": "\u5B50",
  "\u672A": "\u5B50",
  "\u7533": "\u536F",
  "\u9149": "\u536F",
  "\u620C": "\u536F",
  "\u4EA5": "\u5348",
  "\u5B50": "\u5348",
  "\u4E11": "\u5348"
};

// src/liuren/shenSha.ts
var getOffsetBranch = (startBranch, offset) => {
  const idx = DiZhiArray.indexOf(startBranch);
  if (idx === -1) return "";
  let newIdx = (idx + offset) % 12;
  if (newIdx < 0) newIdx += 12;
  return DiZhiArray[newIdx];
};
var seasons = {
  "\u5BC5": "\u6625",
  "\u536F": "\u6625",
  "\u8FB0": "\u6625",
  "\u5DF3": "\u590F",
  "\u5348": "\u590F",
  "\u672A": "\u590F",
  "\u7533": "\u79CB",
  "\u9149": "\u79CB",
  "\u620C": "\u79CB",
  "\u4EA5": "\u51AC",
  "\u5B50": "\u51AC",
  "\u4E11": "\u51AC"
};
var getMonthOrdinal = (mb) => {
  const monthOrdinals = ["\u5BC5", "\u536F", "\u8FB0", "\u5DF3", "\u5348", "\u672A", "\u7533", "\u9149", "\u620C", "\u4EA5", "\u5B50", "\u4E11"];
  return monthOrdinals.indexOf(mb);
};
var rules = [
  // --- Year Sha (岁煞) ---
  { name: "\u592A\u5C81", type: "Year", description: "\u4E3B\u4E00\u5E74\u5409\u51F6\uFF0C\u4EE3\u8868\u541B\u738B\u3001\u5BB6\u957F\u3001\u7B2C\u4E00\u53F7\u4EBA\u7269\uFF0C\u7EDF\u5236\u8BF8\u715E\u3002", calc: (yB) => yB },
  { name: "\u5C81\u7834", type: "Year", description: "\u53C8\u540D\u5927\u8017\uFF0C\u4E3B\u7834\u574F\u3001\u7834\u8017\u8D22\u7269\u3002\u4EE3\u8868\u5BB0\u76F8\u3001\u4E8C\u53F7\u4EBA\u7269\u3002", calc: (yB) => getOffsetBranch(yB, 6) },
  { name: "\u75C5\u7B26", type: "Year", description: "\u4EE3\u8868\u65E7\u4E8B\u3001\u75C5\u6C14\uFF0C\u4E5F\u79F0\u592A\u4E0A\u7687\u3002", calc: (yB) => getOffsetBranch(yB, -1) },
  { name: "\u4E27\u95E8", type: "Year", description: "\u4E3B\u4E27\u4E8B\u3002", calc: (yB) => getOffsetBranch(yB, 2) },
  { name: "\u540A\u5BA2", type: "Year", description: "\u4E3B\u540A\u4E27\u4E4B\u4E8B\u3002", calc: (yB) => getOffsetBranch(yB, -2) },
  { name: "\u5B98\u7B26", type: "Year", description: "\u4E3B\u5B98\u53F8\u8BCD\u8BBC\u3002", calc: (yB) => getOffsetBranch(yB, 4) },
  { name: "\u767D\u864E", type: "Year", description: "\u4E3B\u8840\u5149\u51F6\u4E27\u3002", calc: (yB) => getOffsetBranch(yB, -4) },
  { name: "\u5C81\u5893", type: "Year", description: "\u4E3B\u575F\u5893\u3001\u75C5\u8BBC\u3001\u5B85\u707E\u3002", calc: (yB) => getOffsetBranch(yB, -5) },
  { name: "\u5C81\u5211", type: "Year", description: "\u4E3B\u5211\u4F24\u3001\u5B98\u8BBC\u3002", calc: (yB) => {
    const xingMap = {
      "\u5B50": "\u536F",
      "\u536F": "\u5B50",
      "\u5BC5": "\u5DF3",
      "\u5DF3": "\u7533",
      "\u7533": "\u5BC5",
      "\u4E11": "\u620C",
      "\u620C": "\u672A",
      "\u672A": "\u4E11",
      "\u8FB0": "\u8FB0",
      "\u5348": "\u5348",
      "\u9149": "\u9149",
      "\u4EA5": "\u4EA5"
    };
    return xingMap[yB] || "";
  } },
  { name: "\u5929\u5EAD", type: "Year", description: "\u4E3B\u671D\u5802\u4E4B\u4E8B\u3002", calc: (yB, mB, dS, dB, yS) => {
    const luMap = {
      "\u7532": "\u5BC5",
      "\u4E59": "\u536F",
      "\u4E19": "\u5DF3",
      "\u4E01": "\u5348",
      "\u620A": "\u5DF3",
      "\u5DF1": "\u5348",
      "\u5E9A": "\u7533",
      "\u8F9B": "\u9149",
      "\u58EC": "\u4EA5",
      "\u7678": "\u5B50"
    };
    const lu = luMap[yS];
    if (!lu) return "";
    return getOffsetBranch(lu, -1);
  } },
  // --- Season Sha (季煞) ---
  { name: "\u5929\u559C", type: "Season", description: "\u4E3B\u559C\u5E86\u3001\u6069\u6CFD\u3001\u80CE\u4EA7\u3001\u5B98\u8FC1\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u620C";
    if (s === "\u590F") return "\u4E11";
    if (s === "\u79CB") return "\u8FB0";
    if (s === "\u51AC") return "\u672A";
    return "";
  } },
  { name: "\u7687\u4E66", type: "Season", description: "\u4E3B\u79D1\u540D\u3001\u5B98\u804C\u3001\u8003\u8BD5\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u5BC5";
    if (s === "\u590F") return "\u5DF3";
    if (s === "\u79CB") return "\u7533";
    if (s === "\u51AC") return "\u4EA5";
    return "";
  } },
  { name: "\u5B64\u8FB0", type: "Season", description: "\u4E3B\u5B64\u72EC\u3001\u5A5A\u59FB\u4E0D\u5229\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u5DF3";
    if (s === "\u590F") return "\u7533";
    if (s === "\u79CB") return "\u4EA5";
    if (s === "\u51AC") return "\u5BC5";
    return "";
  } },
  { name: "\u5BE1\u5BBF", type: "Season", description: "\u4E3B\u5B64\u72EC\u3001\u5A5A\u59FB\u4E0D\u5229\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u4E11";
    if (s === "\u590F") return "\u8FB0";
    if (s === "\u79CB") return "\u672A";
    if (s === "\u51AC") return "\u620C";
    return "";
  } },
  { name: "\u5929\u8D66", type: "Season", description: "\u4E3B\u5B98\u8BBC\u89E3\u8131\u3001\u514D\u7F6A\u3001\u6069\u8D66\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u620A\u5BC5";
    if (s === "\u590F") return "\u7532\u5348";
    if (s === "\u79CB") return "\u620A\u7533";
    if (s === "\u51AC") return "\u7532\u5B50";
    return "";
  } },
  { name: "\u56DB\u5E9F", type: "Season", description: "\u4E3B\u767E\u4E8B\u65E0\u6210\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u9149";
    if (s === "\u590F") return "\u5B50";
    if (s === "\u79CB") return "\u536F";
    if (s === "\u51AC") return "\u5348";
    return "";
  } },
  { name: "\u4E27\u8F66", type: "Season", description: "\u4E3B\u75C5\u6B7B\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u9149";
    if (s === "\u590F") return "\u5B50";
    if (s === "\u79CB") return "\u536F";
    if (s === "\u51AC") return "\u5348";
    return "";
  } },
  { name: "\u706B\u9B3C", type: "Season", description: "\u4E3B\u706B\u707E\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u5348";
    if (s === "\u590F") return "\u9149";
    if (s === "\u79CB") return "\u5B50";
    if (s === "\u51AC") return "\u536F";
    return "";
  } },
  { name: "\u6D74\u76C6", type: "Season", description: "\u5C0F\u513F\u5360\u75C5\u5FCC\u4E4B\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u8FB0";
    if (s === "\u590F") return "\u672A";
    if (s === "\u79CB") return "\u620C";
    if (s === "\u51AC") return "\u4E11";
    return "";
  } },
  { name: "\u6E38\u795E", type: "Season", description: "\u4E3B\u98D8\u8361\u3001\u4E0D\u7A33\u5B9A\u3002", calc: (yB, mB) => {
    const s = seasons[mB];
    if (s === "\u6625") return "\u4E11";
    if (s === "\u590F") return "\u5B50";
    if (s === "\u79CB") return "\u4EA5";
    if (s === "\u51AC") return "\u620C";
    return "";
  } },
  // --- Month Sha (月煞) ---
  { name: "\u6708\u5FB7", type: "Month", description: "\u4E3B\u5316\u51F6\u4E3A\u5409\uFF0C\u9022\u51F6\u5316\u5409\u3002", calc: (yB, mB) => {
    if (["\u5BC5", "\u5348", "\u620C"].includes(mB)) return "\u5DF3";
    if (["\u4EA5", "\u536F", "\u672A"].includes(mB)) return "\u5BC5";
    if (["\u7533", "\u5B50", "\u8FB0"].includes(mB)) return "\u4EA5";
    if (["\u5DF3", "\u9149", "\u4E11"].includes(mB)) return "\u7533";
    return "";
  } },
  { name: "\u6708\u5408", type: "Month", description: "\u4E3B\u987A\u5408\u5409\u5E86\u3002", calc: (yB, mB) => {
    const heMap = {
      "\u5BC5": "\u4EA5",
      "\u4EA5": "\u5BC5",
      "\u536F": "\u620C",
      "\u620C": "\u536F",
      "\u8FB0": "\u9149",
      "\u9149": "\u8FB0",
      "\u5DF3": "\u7533",
      "\u7533": "\u5DF3",
      "\u5348": "\u672A",
      "\u672A": "\u5348",
      "\u5B50": "\u4E11",
      "\u4E11": "\u5B50"
    };
    return heMap[mB] || "";
  } },
  { name: "\u751F\u6C14", type: "Month", description: "\u4E3B\u751F\u53D1\u3001\u89E3\u51F6\u589E\u5409\uFF0C\u6210\u5C31\u65B0\u4E8B\u3002", calc: (yB, mB) => getOffsetBranch(mB, -2) },
  { name: "\u6B7B\u6C14", type: "Month", description: "\u4E3B\u6B7B\u4E27\u3001\u75C5\u8BBC\u3001\u6B7B\u4EA1\u4E4B\u60CA\u3002", calc: (yB, mB) => getOffsetBranch(mB, 4) },
  { name: "\u6B7B\u795E", type: "Month", description: "\u4E3B\u6B7B\u4E27\u3001\u4F24\u6B8B\u3002", calc: (yB, mB) => getOffsetBranch(mB, 3) },
  // +4 -1 = +3
  { name: "\u6708\u7834", type: "Month", description: "\u4E3B\u7834\u574F\u3001\u79BB\u6563\u3001\u4E8B\u4F53\u4E0D\u6210\u3002", calc: (yB, mB) => getOffsetBranch(mB, 6) },
  { name: "\u5929\u5FB7", type: "Month", description: "\u4E3B\u5316\u51F6\u4E3A\u5409\uFF0C\u8D35\u4EBA\u6276\u52A9\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u672A", "\u7533", "\u4EA5", "\u620C", "\u4EA5", "\u5BC5", "\u4E11", "\u5BC5", "\u5DF3", "\u8FB0", "\u5DF3", "\u7533"];
    return map[ord] || "";
  } },
  { name: "\u5929\u8BCF", type: "Month", description: "\u4E3B\u541B\u547D\u3001\u5347\u8FC1\u3001\u8BCF\u4E66\u4E4B\u559C\u3002", calc: (yB, mB) => getOffsetBranch(mB, -3) },
  { name: "\u5929\u533B", type: "Month", description: "\u4E3B\u6CBB\u75C5\u3002", calc: (yB, mB) => getOffsetBranch(mB, 2) },
  { name: "\u5929\u9A6C", type: "Month", description: "\u4E3B\u8FC1\u52A8\u3001\u8BCF\u547D\u3001\u51FA\u884C\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const horses = ["\u5348", "\u7533", "\u620C", "\u5B50", "\u5BC5", "\u8FB0"];
    return horses[ord % 6];
  } },
  { name: "\u5929\u8D22", type: "Month", description: "\u4E3B\u6C42\u8D22\u3001\u8C0B\u671B\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const money = ["\u8FB0", "\u5348", "\u7533", "\u620C", "\u5B50", "\u5BC5"];
    return money[ord % 6];
  } },
  { name: "\u6210\u795E", type: "Month", description: "\u4E3B\u8C0B\u671B\u4E8B\u6210\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u5DF3", "\u7533", "\u4EA5", "\u5BC5"];
    return map[ord % 4];
  } },
  { name: "\u7687\u6069", type: "Month", description: "\u4E3B\u7687\u6069\u5E87\u62A4\u3001\u5B98\u804C\u5347\u8FC1\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u672A", "\u9149", "\u4EA5", "\u4E11", "\u536F", "\u5DF3"];
    return map[ord % 6];
  } },
  { name: "\u5929\u89E3", type: "Month", description: "\u4E3B\u5B98\u8BBC\u89E3\u8131\u3001\u514D\u7F6A\u3001\u6D88\u707E\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u7533", "\u7533", "\u620C", "\u620C", "\u5B50", "\u5B50", "\u5BC5", "\u5BC5", "\u8FB0", "\u8FB0", "\u5348", "\u5348"];
    return map[ord];
  } },
  { name: "\u6708\u538C", type: "Month", description: "\u4E3B\u59A8\u5A5A\u5A36\u3001\u8BE1\u602A\u3001\u76D7\u8D3C\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const idx = (11 - ord + 12) % 12;
    return DiZhiArray[idx];
  } },
  { name: "\u5927\u715E", type: "Month", description: "\u4E3B\u707E\u901F\u3001\u5BB6\u957F\u51F6\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u5348", "\u536F", "\u5B50", "\u9149"];
    return map[ord % 4];
  } },
  { name: "\u5C0F\u715E", type: "Month", description: "\u4E3B\u5C0F\u513F\u707E\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u4E11", "\u620C", "\u672A", "\u8FB0"];
    return map[ord % 4];
  } },
  { name: "\u5929\u9B3C", type: "Month", description: "\u4E3B\u5175\u4EA1\u3001\u4EA7\u6B7B\u3001\u75AB\u6C14\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u9149", "\u5348", "\u536F", "\u5B50"];
    return map[ord % 4];
  } },
  { name: "\u5929\u5211", type: "Month", description: "\u4E3B\u5B98\u8BBC\u3001\u56DA\u7CFB\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u5BC5", "\u8FB0", "\u5348", "\u7533", "\u620C", "\u5B50"];
    return map[ord % 6];
  } },
  { name: "\u5929\u72F1", type: "Month", description: "\u4E3B\u7CFB\u5F79\u3001\u7262\u72F1\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u4EA5", "\u7533", "\u5DF3", "\u5BC5"];
    return map[ord % 4];
  } },
  { name: "\u52AB\u715E", type: "Month", description: "\u4E3B\u52AB\u63A0\u3001\u963B\u788D\u3001\u707E\u75C5\u3002", calc: (yB, mB) => {
    if (["\u7533", "\u5B50", "\u8FB0"].includes(mB)) return "\u5DF3";
    if (["\u5BC5", "\u5348", "\u620C"].includes(mB)) return "\u4EA5";
    if (["\u5DF3", "\u9149", "\u4E11"].includes(mB)) return "\u5BC5";
    if (["\u4EA5", "\u536F", "\u672A"].includes(mB)) return "\u7533";
    return "";
  } },
  { name: "\u4EA1\u795E", type: "Month", description: "\u4E3B\u5931\u7269\u3001\u75C5\u8BBC\u3001\u6B7B\u4EA1\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u5DF3", "\u5BC5", "\u4EA5", "\u7533"];
    return map[ord % 4];
  } },
  { name: "\u8840\u652F", type: "Month", description: "\u4E3B\u8840\u5149\u3001\u624B\u672F\u3002", calc: (yB, mB) => getOffsetBranch(mB, -1) },
  { name: "\u8840\u5FCC", type: "Month", description: "\u4E3B\u8840\u5149\u3001\u6D41\u8840\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    const map = ["\u4E11", "\u672A", "\u5BC5", "\u7533", "\u536F", "\u9149", "\u8FB0", "\u620C", "\u5DF3", "\u4EA5", "\u5348", "\u5B50"];
    return map[ord] || "";
  } },
  { name: "\u5929\u8D3C", type: "Month", description: "\u4E3B\u76D7\u8D3C\u3001\u9057\u5931\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB) + 1;
    if (ord % 2 !== 0) {
      const map = ["\u8FB0", "\u5348", "\u7533", "\u620C", "\u5B50", "\u5BC5"];
      return map[Math.floor(ord / 2)];
    } else {
      const map = ["\u9149", "\u4EA5", "\u4E11", "\u536F", "\u5DF3", "\u672A"];
      return map[Math.floor(ord / 2) - 1];
    }
  } },
  { name: "\u5978\u95E8", type: "Month", description: "\u4E3B\u5978\u6DEB\u4E0D\u6B63\u3002", calc: (yB, mB) => {
    const ord = getMonthOrdinal(mB);
    if (ord === -1) return "";
    const map = ["\u7533", "\u4EA5", "\u5BC5", "\u5DF3"];
    return map[ord % 4];
  } },
  // --- Day Sha (日煞) ---
  { name: "\u65E5\u5FB7", type: "Day", description: "\u4E3B\u5409\u5E86\u3001\u8D35\u4EBA\u3002", calc: (yB, mB, dS) => riDe[dS] || "" },
  { name: "\u65E5\u7984", type: "Day", description: "\u4E3B\u8D22\u7984\u3001\u98DF\u7984\u3002", calc: (yB, mB, dS) => riLu[dS] || "" },
  { name: "\u7834\u788E", type: "Day", description: "\u4E3B\u7834\u8D25\u3001\u635F\u8017\u3002", calc: (yB, mB, dS, dB) => PoSui[dB] || "" }
];
var getShenSha = (date) => {
  const baziArr = date.bazi.split(" ");
  const yearPillar = baziArr[0];
  const yearStem = yearPillar.substring(0, 1);
  const yearBranch = yearPillar.substring(1, 2);
  const monthPillar = baziArr[1];
  const monthBranch = monthPillar.substring(1, 2);
  const dayPillar = baziArr[2];
  const dayStem = dayPillar.substring(0, 1);
  const dayBranch = dayPillar.substring(1, 2);
  const result = [];
  for (const rule of rules) {
    const val = rule.calc(yearBranch, monthBranch, dayStem, dayBranch, yearStem);
    if (val) {
      result.push({
        name: rule.name,
        value: val,
        description: rule.description
      });
    }
  }
  return result;
};

// src/maps/shenJiang.ts
var ShenJiangArray = ["\u8D35\u4EBA", "\u817E\u86C7", "\u6731\u96C0", "\u516D\u5408", "\u52FE\u9648", "\u9752\u9F99", "\u5929\u7A7A", "\u767D\u864E", "\u592A\u5E38", "\u7384\u6B66", "\u592A\u9634", "\u5929\u540E"];
var YangGui = {
  "\u7532": "\u4E11",
  "\u620A": "\u4E11",
  "\u5E9A": "\u4E11",
  "\u4E59": "\u5B50",
  "\u5DF1": "\u5B50",
  "\u4E19": "\u4EA5",
  "\u4E01": "\u4EA5",
  "\u58EC": "\u5DF3",
  "\u7678": "\u5DF3",
  "\u8F9B": "\u5348"
};
var YinGui = {
  "\u7532": "\u672A",
  "\u620A": "\u672A",
  "\u5E9A": "\u672A",
  "\u4E59": "\u7533",
  "\u5DF1": "\u7533",
  "\u4E19": "\u9149",
  "\u4E01": "\u9149",
  "\u58EC": "\u536F",
  "\u7678": "\u536F",
  "\u8F9B": "\u5BC5"
};
var YinYang = {
  "\u536F": "\u9633",
  "\u8FB0": "\u9633",
  "\u5DF3": "\u9633",
  "\u5348": "\u9633",
  "\u672A": "\u9633",
  "\u7533": "\u9633",
  "\u9149": "\u9634",
  "\u620C": "\u9634",
  "\u4EA5": "\u9634",
  "\u5B50": "\u9634",
  "\u4E11": "\u9634",
  "\u5BC5": "\u9634"
};
var ShunNi = {
  "\u4EA5": "\u987A",
  "\u5B50": "\u987A",
  "\u4E11": "\u987A",
  "\u5BC5": "\u987A",
  "\u536F": "\u987A",
  "\u8FB0": "\u987A",
  "\u5DF3": "\u9006",
  "\u5348": "\u9006",
  "\u672A": "\u9006",
  "\u7533": "\u9006",
  "\u9149": "\u9006",
  "\u620C": "\u9006"
};

// src/liuren/yinYangGuiRen.ts
var arrangeTianJiang = (tianPan, guiZhi) => {
  const tianJiang = {
    0: "",
    1: "",
    2: "",
    3: "",
    4: "",
    5: "",
    6: "",
    7: "",
    8: "",
    9: "",
    10: "",
    11: ""
  };
  let guiIndex = 0;
  for (let i = 0; i < 12; i++) {
    if (guiZhi === tianPan[i]) {
      guiIndex = i;
      break;
    }
  }
  const shunNi = ShunNi[DiZhiArray[guiIndex]];
  for (let i = 0; i < 12; i++) {
    let index = guiIndex;
    if (shunNi === "\u987A") {
      index = index + i;
    } else {
      index = index - i;
    }
    if (index > 11) index -= 12;
    if (index < 0) index += 12;
    tianJiang[index] = ShenJiangArray[i];
  }
  return tianJiang;
};
var getYinYangGuiRen = (date, tianDiPan) => {
  const riGan = date.bazi.split(" ")[2].substring(0, 1);
  const tianPan = tianDiPan.\u5929\u76D8;
  const yangGuiZhi = YangGui[riGan];
  const yinGuiZhi = YinGui[riGan];
  return {
    \u9633\u8D35\u4EBA: arrangeTianJiang(tianPan, yangGuiZhi),
    \u9634\u8D35\u4EBA: arrangeTianJiang(tianPan, yinGuiZhi)
  };
};

// src/liuren/tianDiPan.ts
var getTianDiPan = (date) => {
  const result = {
    "\u5730\u76D8": {
      0: "\u5B50",
      1: "\u4E11",
      2: "\u5BC5",
      3: "\u536F",
      4: "\u8FB0",
      5: "\u5DF3",
      6: "\u5348",
      7: "\u672A",
      8: "\u7533",
      9: "\u9149",
      10: "\u620C",
      11: "\u4EA5"
    },
    "\u5929\u76D8": {
      0: "",
      1: "",
      2: "",
      3: "",
      4: "",
      5: "",
      6: "",
      7: "",
      8: "",
      9: "",
      10: "",
      11: ""
    },
    "\u5929\u5C06": {
      0: "",
      1: "",
      2: "",
      3: "",
      4: "",
      5: "",
      6: "",
      7: "",
      8: "",
      9: "",
      10: "",
      11: ""
    }
  };
  const tianPan = {
    0: "",
    1: "",
    2: "",
    3: "",
    4: "",
    5: "",
    6: "",
    7: "",
    8: "",
    9: "",
    10: "",
    11: ""
  };
  const yueJiang = date.yuejiang;
  const shiChen = date.bazi.split(" ")[3].substring(1, 2);
  const shiChenIndex = DiZhiArray.indexOf(shiChen);
  const yueJiangIndex = DiZhiArray.indexOf(yueJiang);
  for (let i = 0; i < 12; i++) {
    let indexChenTemp = shiChenIndex + i;
    let indexYueJiangTemp = yueJiangIndex + i;
    if (indexChenTemp > 11) {
      indexChenTemp = indexChenTemp - 12;
    }
    if (indexYueJiangTemp > 11) {
      indexYueJiangTemp = indexYueJiangTemp - 12;
    }
    tianPan[indexChenTemp] = DiZhiArray[indexYueJiangTemp];
  }
  result.\u5929\u76D8 = tianPan;
  const tianJiang = {
    0: "",
    1: "",
    2: "",
    3: "",
    4: "",
    5: "",
    6: "",
    7: "",
    8: "",
    9: "",
    10: "",
    11: ""
  };
  const yinYang = YinYang[shiChen];
  let guiZhi = "";
  if (yinYang === "\u9633") {
    guiZhi = YangGui[date.bazi.split(" ")[2].substring(0, 1)];
  } else {
    guiZhi = YinGui[date.bazi.split(" ")[2].substring(0, 1)];
  }
  let guiIndex = 0;
  for (let i = 0; i < 12; i++) {
    if (guiZhi == tianPan[i]) {
      guiIndex = i;
      break;
    }
  }
  const shunNi = ShunNi[DiZhiArray[guiIndex]];
  for (let i = 0; i < 12; i++) {
    let index = guiIndex;
    if (shunNi === "\u987A") {
      index = index + i;
    } else {
      index = index - i;
    }
    if (index > 11) {
      index = index - 12;
    }
    if (index < 0) {
      index = index + 12;
    }
    tianJiang[index] = ShenJiangArray[i];
  }
  result.\u5929\u5C06 = tianJiang;
  return result;
};
var getGongIndex = (tiandipan, tian) => {
  const tianPan = tiandipan.\u5929\u76D8;
  for (let i = 0; i < 12; i++) {
    if (tianPan[i] === tian) {
      return i;
    }
  }
  return 0;
};

// src/jinKouJue/shenSha.ts
var getOffsetZhi = (startZhi, offset) => {
  let idx = DiZhiArray.indexOf(startZhi);
  if (idx === -1) return "";
  let targetIdx = (idx + offset) % 12;
  if (targetIdx < 0) targetIdx += 12;
  return DiZhiArray[targetIdx];
};
var getPositions = (target, siWei) => {
  const pos = [];
  if (siWei.renYuan.ganZhi.includes(target)) pos.push("\u4EBA\u5143");
  if (siWei.guiShen.ganZhi.includes(target)) pos.push("\u8D35\u795E");
  if (siWei.jiangShen.ganZhi.includes(target)) pos.push("\u5C06\u795E");
  if (siWei.diFen.ganZhi.includes(target)) pos.push("\u5730\u5206");
  return pos;
};
var getShenSha2 = (result) => {
  const shenShaList = [];
  const dateInfo = result.date;
  const siWei = result.siWei;
  const baziParts = dateInfo.bazi.split(" ");
  const yearGanZhi = baziParts[0];
  const monthGanZhi = baziParts[1];
  const dayGanZhi = baziParts[2];
  const yearZhi = yearGanZhi[1];
  const monthZhi = monthGanZhi[1];
  const dayGan = dayGanZhi[0];
  const dayZhi = dayGanZhi[1];
  const addIfIn = (name, val, desc, type) => {
    if (!val) return;
    const pos = getPositions(val, siWei);
    if (pos.length > 0) {
      shenShaList.push({ name, value: val, position: pos, description: desc, type });
    }
  };
  const tianDeVal = TianDe[monthZhi];
  addIfIn("\u5929\u5FB7", tianDeVal, "\u4E3B\u5316\u89E3\u767E\u7978\uFF0C\u9022\u51F6\u5316\u5409", "\u5409");
  if (tianDeVal) {
    let tianDeHe = TianGanArray.includes(tianDeVal) ? TianGanWuHe[tianDeVal] : LiuHe[tianDeVal];
    addIfIn("\u5929\u5FB7\u5408", tianDeHe, "\u4E3B\u89E3\u767E\u7978\uFF0C\u5409\u5E86\uFF0C\u5316\u51F6\u89E3\u5FE7", "\u5409");
  }
  const yueDeVal = YueDe[monthZhi];
  addIfIn("\u6708\u5FB7", yueDeVal, "\u4E3B\u548C\u7766\uFF0C\u4E07\u4E8B\u987A\u8FBE", "\u5409");
  if (yueDeVal) {
    addIfIn("\u6708\u5FB7\u5408", TianGanWuHe[yueDeVal], "\u4F5C\u7528\u540C\u6708\u5FB7\uFF0C\u5409\u5E86\u7A0D\u6B21", "\u5409");
  }
  const monthIdx = DiZhiArray.indexOf(monthZhi);
  let season = "";
  if (monthIdx >= 2 && monthIdx <= 4) season = "\u6625";
  else if (monthIdx >= 5 && monthIdx <= 7) season = "\u590F";
  else if (monthIdx >= 8 && monthIdx <= 10) season = "\u79CB";
  else season = "\u51AC";
  let sheVals = [];
  if (season === "\u6625" && dayGanZhi === "\u620A\u5BC5") sheVals = ["\u620A", "\u5BC5"];
  if (season === "\u590F" && dayGanZhi === "\u7532\u5348") sheVals = ["\u7532", "\u5348"];
  if (season === "\u79CB" && dayGanZhi === "\u620A\u7533") sheVals = ["\u620A", "\u7533"];
  if (season === "\u51AC" && dayGanZhi === "\u7532\u5B50") sheVals = ["\u7532", "\u5B50"];
  sheVals.forEach((v) => addIfIn("\u5929\u8D66", v, "\u4E3B\u89E3\u5211\u7981\u3001\u5B98\u53F8\u3001\u5371\u9669\u4E4B\u707E", "\u5409"));
  addIfIn("\u5929\u559C", TianXi[monthZhi], "\u4E3B\u5BB6\u4E2D\u559C\u5E86\u3001\u5A5A\u59FB\u3001\u8FDB\u8D22", "\u5409");
  addIfIn("\u5929\u533B", TianYi[monthZhi], "\u95EE\u75C5\u5FE7\u4E2D\u6709\u4E50\uFF0C\u5F97\u826F\u533B", "\u5409");
  addIfIn("\u5929\u9A6C", TianMa2[monthZhi], "\u4E3B\u529E\u4E8B\u8FC5\u901F\uFF0C\u9003\u4EA1\u8FDC\u53BB\u4E0D\u5F52", "\u5409");
  addIfIn("\u9A7F\u9A6C", YiMa[dayZhi], "\u4E3B\u6C42\u4E8B\u8FC5\u901F\uFF0C\u5347\u8FC1\u8FDC\u884C", "\u5409");
  const lessonStems = /* @__PURE__ */ new Set();
  const checkStem = (gz) => {
    if (gz.length > 0 && TianGanArray.includes(gz[0])) lessonStems.add(gz[0]);
  };
  checkStem(siWei.renYuan.ganZhi);
  checkStem(siWei.guiShen.ganZhi);
  checkStem(siWei.jiangShen.ganZhi);
  const addSanQi = (stems, name) => {
    if (stems.every((s) => lessonStems.has(s))) {
      stems.forEach((s) => addIfIn(name, s, "\u5229\u89C1\u5927\u4EBA\uFF0C\u767E\u4E8B\u5409\u660C", "\u5409"));
    }
  };
  addSanQi(["\u7532", "\u620A", "\u5E9A"], "\u5929\u4E09\u5947");
  addSanQi(["\u4E59", "\u4E19", "\u4E01"], "\u5730\u4E09\u5947");
  addSanQi(["\u58EC", "\u7678", "\u8F9B"], "\u4EBA\u4E09\u5947");
  addIfIn("\u751F\u6C14", ShengQi[monthZhi], "\u7EDD\u5904\u9022\u751F\uFF0C\u5F00\u8F9F\u65B0\u4E8B\u4E1A", "\u5409");
  if (siWei.renYuan.ganZhi === "\u7532") {
    shenShaList.push({ name: "\u516D\u7532", value: "\u7532", position: ["\u4EBA\u5143"], description: "\u4E3B\u6709\u4E0D\u6D4B\u4E4B\u559C\uFF0C\u4E3A\u5934\u76EE", type: "\u5409" });
  }
  addIfIn("\u52AB\u715E", JieSha[dayZhi], "\u5E38\u4EBA\u4E3B\u51F6\u4F24\u3001\u5B98\u53F8", "\u51F6");
  const jieMingMap = {
    "\u7532": ["\u7533", "\u9149"],
    "\u5DF1": ["\u7533", "\u9149"],
    "\u4E59": ["\u5348", "\u672A"],
    "\u5E9A": ["\u5348", "\u672A"],
    "\u4E19": ["\u8FB0", "\u5DF3"],
    "\u8F9B": ["\u8FB0", "\u5DF3"],
    "\u4E01": ["\u5BC5", "\u536F"],
    "\u58EC": ["\u5BC5", "\u536F"],
    "\u620A": ["\u5B50", "\u4E11"],
    "\u7678": ["\u5B50", "\u4E11"]
  };
  (jieMingMap[dayGan] || []).forEach((v) => addIfIn("\u622A\u547D\u707E\u715E", v, "\u6C42\u8C0B\u4E0D\u901A\uFF0C\u51FA\u884C\u53D7\u963B", "\u51F6"));
  const wuGuiMap = {
    "\u7532": ["\u5DF3", "\u5348"],
    "\u5DF1": ["\u5DF3", "\u5348"],
    "\u4E59": ["\u5BC5", "\u536F"],
    "\u5E9A": ["\u5BC5", "\u536F"],
    "\u4E19": ["\u5B50", "\u4E11"],
    "\u8F9B": ["\u5B50", "\u4E11"],
    "\u4E01": ["\u620C", "\u4EA5"],
    "\u58EC": ["\u620C", "\u4EA5"],
    "\u620A": ["\u7533", "\u9149"],
    "\u7678": ["\u7533", "\u9149"]
  };
  (wuGuiMap[dayGan] || []).forEach((v) => addIfIn("\u4E94\u9B3C", v, "\u635F\u8D22\u3001\u5B98\u53F8\u3001\u8F66\u7978", "\u51F6"));
  addIfIn("\u4E27\u95E8", getOffsetZhi(yearZhi, 2), "\u4E3B\u51F6\u4E27\u3001\u5B5D\u670D\u3001\u54ED\u6CE3", "\u51F6");
  addIfIn("\u540A\u5BA2", getOffsetZhi(yearZhi, -2), "\u4E3B\u9634\u79C1\u3001\u51F6\u4F24\u3001\u4EB2\u670B\u51F6\u4E27", "\u51F6");
  addIfIn("\u4E27\u8F66", SangChe[monthZhi], "\u4E3B\u75C5\u707E\u3001\u8F66\u7978\u3001\u8840\u5149", "\u51F6");
  addIfIn("\u5929\u9B3C", TianGui[monthZhi], "\u4E3B\u51F6\u707E\uFF0C\u9B3C\u53D8", "\u51F6");
  const isYangMonth = ["\u5B50", "\u5BC5", "\u8FB0", "\u5348", "\u7533", "\u620C"].includes(monthZhi);
  addIfIn("\u706D\u95E8", getOffsetZhi(monthZhi, isYangMonth ? -3 : 3), "\u5FCC\u8FC1\u5C45\u3001\u5AC1\u5A36\uFF0C\u4E3B\u75C5\u707E", "\u51F6");
  const tianLuo = getOffsetZhi(dayZhi, 1);
  addIfIn("\u5929\u7F57", tianLuo, "\u4E3B\u7262\u72F1\u5B98\u53F8", "\u51F6");
  addIfIn("\u5730\u7F51", LiuChong[tianLuo], "\u4E3B\u7262\u72F1\u5B98\u53F8", "\u51F6");
  const checkStack = (bottom, top, name) => {
    if (siWei.diFen.ganZhi === bottom && siWei.jiangShen.ganZhi.endsWith(top)) {
      shenShaList.push({ name, value: `${bottom}${top}`, position: ["\u5730\u5206", "\u5C06\u795E"], description: "\u5173\u8282\u4E0D\u901A\uFF0C\u56DA\u7981", type: "\u51F6" });
    }
    if (siWei.jiangShen.ganZhi.endsWith(bottom) && siWei.guiShen.ganZhi.endsWith(top)) {
      shenShaList.push({ name, value: `${bottom}${top}`, position: ["\u5C06\u795E", "\u8D35\u795E"], description: "\u5173\u8282\u4E0D\u901A\uFF0C\u56DA\u7981", type: "\u51F6" });
    }
  };
  checkStack("\u9149", "\u5BC5", "\u5173");
  checkStack("\u536F", "\u620C", "\u9694");
  checkStack("\u536F", "\u7533", "\u9501");
  const checkPair = (z1, z2, name) => {
    const p1 = getPositions(z1, siWei), p2 = getPositions(z2, siWei);
    if (p1.length > 0 && p2.length > 0) {
      shenShaList.push({ name, value: `${z1}${z2}`, position: [.../* @__PURE__ */ new Set([...p1, ...p2])], description: "\u4E3B\u529E\u4E8B\u96BE\u6210\uFF0C\u52B3\u800C\u65E0\u529F", type: "\u51F6" });
    }
  };
  checkPair("\u5BC5", "\u9149", "\u91D1\u7EDD");
  checkPair("\u536F", "\u7533", "\u6728\u7EDD");
  checkPair("\u5348", "\u4EA5", "\u6C34\u7EDD");
  checkPair("\u5B50", "\u5DF3", "\u706B\u7EDD");
  const hasWuXing = (wx) => [siWei.renYuan, siWei.guiShen, siWei.jiangShen, siWei.diFen].some((p) => p.wuXing === wx);
  if (getPositions("\u9149", siWei).length > 0 && (hasWuXing("\u6C34") || hasWuXing("\u571F"))) addIfIn("\u56DB\u8D25", "\u9149", "\u4E3B\u62D8\u7981\u3001\u53E3\u820C", "\u51F6");
  if (getPositions("\u536F", siWei).length > 0 && hasWuXing("\u706B")) addIfIn("\u56DB\u8D25", "\u536F", "\u4E3B\u62D8\u7981\u3001\u53E3\u820C", "\u51F6");
  if (getPositions("\u5B50", siWei).length > 0 && hasWuXing("\u6728")) addIfIn("\u56DB\u8D25", "\u5B50", "\u4E3B\u62D8\u7981\u3001\u53E3\u820C", "\u51F6");
  if (getPositions("\u5348", siWei).length > 0 && hasWuXing("\u91D1")) addIfIn("\u56DB\u8D25", "\u5348", "\u4E3B\u62D8\u7981\u3001\u53E3\u820C", "\u51F6");
  ["\u5B50"].forEach((v) => {
    const pos = [];
    if (siWei.guiShen.ganZhi.endsWith(v)) pos.push("\u8D35\u795E");
    if (siWei.jiangShen.ganZhi.endsWith(v)) pos.push("\u5C06\u795E");
    if (pos.length > 0) shenShaList.push({ name: "\u5929\u76D7", value: v, position: pos, description: "\u4E3B\u88AB\u76D7\u3001\u5931\u8D22", type: "\u51F6" });
  });
  addIfIn("\u98DE\u5EC9", FeiLian[monthZhi], "\u4E3B\u8FC5\u901F\uFF0C\u975E\u5E38\u60CA\u9A87", "\u51F6");
  addIfIn("\u56DB\u4E18", SiQiu[monthZhi], "\u4E3B\u4E89\u8BBA\u7530\u571F\u575F\u5893", "\u51F6");
  addIfIn("\u56DB\u5893", SiMu[monthZhi], "\u4E3B\u4E89\u8BBC\u575F\u5893\u4E4B\u4E8B", "\u51F6");
  if (JieSha[dayZhi]) {
    addIfIn("\u671B\u95E8\u715E", LiuChong[JieSha[dayZhi]], "\u4E3B\u5984\u60F3\u3001\u7A7A\u60F3", "\u51F6");
  }
  addIfIn("\u75C5\u7B26", getOffsetZhi(yearZhi, -1), "\u4E3B\u5927\u75C5\u3001\u707E\u7978", "\u51F6");
  const isDay = YinYang[baziParts[3][1]] === "\u9633";
  const guiRenZhi = isDay ? YangGui[dayGan] : YinGui[dayGan];
  addIfIn("\u5B98\u7B26", LiuChong[guiRenZhi], "\u4E3B\u5B98\u53F8\u5211\u6CD5", "\u51F6");
  if (siWei.renYuan.ganZhi === "\u4E01") addIfIn("\u516D\u4E01", "\u4E01", "\u4E3B\u5BB6\u4E2D\u4E0D\u5B89\u5B81", "\u51F6");
  const yearLu = riLu[yearGanZhi[0]];
  if (yearLu) addIfIn("\u7984\u5012", getOffsetZhi(yearLu, 1), "\u4E3B\u767E\u4E8B\u4E0D\u987A", "\u51F6");
  const yearMa = YiMa[yearZhi];
  if (yearMa) addIfIn("\u9A6C\u5012", getOffsetZhi(yearMa, 1), "\u4E3B\u767E\u4E8B\u4E0D\u987A", "\u51F6");
  return shenShaList;
};

// src/jinKouJue/index.ts
var getWuShuDun = (dayGan, diZhi) => {
  const startGanMap = {
    "\u7532": "\u7532",
    "\u5DF1": "\u7532",
    "\u4E59": "\u4E19",
    "\u5E9A": "\u4E19",
    "\u4E19": "\u620A",
    "\u8F9B": "\u620A",
    "\u4E01": "\u5E9A",
    "\u58EC": "\u5E9A",
    "\u620A": "\u58EC",
    "\u7678": "\u58EC"
  };
  const startGan = startGanMap[dayGan];
  const startGanIndex = TianGanArray.indexOf(startGan);
  const zhiIndex = DiZhiArray.indexOf(diZhi);
  const targetGanIndex = (startGanIndex + zhiIndex) % 10;
  return TianGanArray[targetGanIndex];
};
var getJiangShen = (yueJiang, timeZhi, diFen) => {
  const yueJiangIdx = DiZhiNumber[yueJiang] - 1;
  const timeZhiIdx = DiZhiNumber[timeZhi] - 1;
  const diFenIdx = DiZhiNumber[diFen] - 1;
  let diff = diFenIdx - timeZhiIdx;
  let jiangShenIdx = (yueJiangIdx + diff) % 12;
  if (jiangShenIdx < 0) jiangShenIdx += 12;
  return DiZhiArray[jiangShenIdx];
};
var getGuiShen = (dayGan, timeZhi, diFen) => {
  const isDay = YinYang[timeZhi] === "\u9633";
  let guiRenStartZhi = "";
  if (isDay) {
    guiRenStartZhi = YangGui[dayGan];
  } else {
    guiRenStartZhi = YinGui[dayGan];
  }
  const shunNi = ShunNi[guiRenStartZhi];
  const isShun = shunNi === "\u987A";
  const startZhiIdx = DiZhiNumber[guiRenStartZhi] - 1;
  const diFenIdx = DiZhiNumber[diFen] - 1;
  let step = 0;
  if (isShun) {
    step = diFenIdx - startZhiIdx;
  } else {
    step = startZhiIdx - diFenIdx;
  }
  step = step % 12;
  if (step < 0) step += 12;
  const shenIndex = step;
  return ShenJiangArray[shenIndex];
};
var WuXingSheng = {
  "\u6728": "\u706B",
  "\u706B": "\u571F",
  "\u571F": "\u91D1",
  "\u91D1": "\u6C34",
  "\u6C34": "\u6728"
};
var WuXingKe = {
  "\u6728": "\u571F",
  "\u571F": "\u6C34",
  "\u6C34": "\u706B",
  "\u706B": "\u91D1",
  "\u91D1": "\u6728"
};
var getShengWo = (wuXing) => {
  for (const [k, v] of Object.entries(WuXingSheng)) {
    if (v === wuXing) return k;
  }
  return "";
};
var getKeWo = (wuXing) => {
  for (const [k, v] of Object.entries(WuXingKe)) {
    if (v === wuXing) return k;
  }
  return "";
};
var getYueLingWuXing = (monthZhi) => {
  const monthZhiToWang = {
    "\u5BC5": "\u6728",
    "\u536F": "\u6728",
    "\u5DF3": "\u706B",
    "\u5348": "\u706B",
    "\u7533": "\u91D1",
    "\u9149": "\u91D1",
    "\u4EA5": "\u6C34",
    "\u5B50": "\u6C34",
    "\u8FB0": "\u571F",
    "\u620C": "\u571F",
    "\u4E11": "\u571F",
    "\u672A": "\u571F"
  };
  return monthZhiToWang[monthZhi] || "";
};
var getWangXiangXiuQiu = (wuXing, monthZhi) => {
  if (!wuXing) return "";
  const yueLingWuXing = getYueLingWuXing(monthZhi);
  if (!yueLingWuXing) return "";
  if (wuXing === yueLingWuXing) return "\u65FA";
  if (wuXing === WuXingSheng[yueLingWuXing]) return "\u76F8";
  if (wuXing === getShengWo(yueLingWuXing)) return "\u4F11";
  if (wuXing === getKeWo(yueLingWuXing)) return "\u56DA";
  if (wuXing === WuXingKe[yueLingWuXing]) return "\u6B7B";
  return "";
};
var getWuXing = (name) => {
  if (WuXing[name]) {
    return WuXing[name];
  }
  const shenMap = {
    "\u8D35\u4EBA": "\u4E11",
    "\u817E\u86C7": "\u5DF3",
    "\u6731\u96C0": "\u5348",
    "\u516D\u5408": "\u536F",
    "\u52FE\u9648": "\u8FB0",
    "\u9752\u9F99": "\u5BC5",
    "\u5929\u7A7A": "\u620C",
    "\u767D\u864E": "\u7533",
    "\u592A\u5E38": "\u672A",
    "\u7384\u6B66": "\u5B50",
    "\u592A\u9634": "\u9149",
    "\u5929\u540E": "\u4EA5"
  };
  if (shenMap[name]) {
    return WuXing[shenMap[name]];
  }
  for (const [zhi, yueName] of Object.entries(YueJiangName)) {
    if (yueName === name) {
      return WuXing[zhi];
    }
  }
  return "";
};
var getShenGanZhi = (name) => {
  const shenMap = {
    "\u8D35\u4EBA": "\u4E11",
    "\u817E\u86C7": "\u5DF3",
    "\u6731\u96C0": "\u5348",
    "\u516D\u5408": "\u536F",
    "\u52FE\u9648": "\u8FB0",
    "\u9752\u9F99": "\u5BC5",
    "\u5929\u7A7A": "\u620C",
    "\u767D\u864E": "\u7533",
    "\u592A\u5E38": "\u672A",
    "\u7384\u6B66": "\u5B50",
    "\u592A\u9634": "\u9149",
    "\u5929\u540E": "\u4EA5"
  };
  return shenMap[name] || name;
};
var getJinKouJueByDateInfo = (dateInfo, diFen) => {
  const dayGan = dateInfo.bazi.split(" ")[2].substring(0, 1);
  const timeZhi = dateInfo.bazi.split(" ")[3].substring(1, 2);
  const monthZhi = dateInfo.bazi.split(" ")[1].substring(1, 2);
  const yueJiang = dateInfo.yuejiang;
  const renYuanName = getWuShuDun(dayGan, diFen);
  const renYuanWuXing = getWuXing(renYuanName);
  const guiShenName = getGuiShen(dayGan, timeZhi, diFen);
  const guiShenZhi = getShenGanZhi(guiShenName);
  const guiShenGan = getWuShuDun(dayGan, guiShenZhi);
  const guiShenWuXing = getWuXing(guiShenName);
  const jiangShenZhi = getJiangShen(yueJiang, timeZhi, diFen);
  const jiangShenName = YueJiangName[jiangShenZhi] || jiangShenZhi;
  const jiangShenGan = getWuShuDun(dayGan, jiangShenZhi);
  const jiangShenWuXing = getWuXing(jiangShenZhi);
  const diFenWuXing = getWuXing(diFen);
  const siWei = {
    renYuan: {
      name: renYuanName,
      ganZhi: renYuanName,
      wuXing: renYuanWuXing,
      wangXiangXiuQiu: getWangXiangXiuQiu(renYuanWuXing, monthZhi)
    },
    guiShen: {
      name: guiShenName,
      ganZhi: guiShenGan + guiShenZhi,
      wuXing: guiShenWuXing,
      wangXiangXiuQiu: getWangXiangXiuQiu(guiShenWuXing, monthZhi)
    },
    jiangShen: {
      name: jiangShenName,
      // 使用别名，如 大吉
      ganZhi: jiangShenGan + jiangShenZhi,
      // 干支
      wuXing: jiangShenWuXing,
      // 五行用地支查更准
      wangXiangXiuQiu: getWangXiangXiuQiu(jiangShenWuXing, monthZhi)
    },
    diFen: {
      name: diFen,
      ganZhi: diFen,
      wuXing: diFenWuXing,
      wangXiangXiuQiu: getWangXiangXiuQiu(diFenWuXing, monthZhi)
    }
  };
  const result = {
    date: dateInfo,
    diFen,
    siWei,
    shenSha: []
  };
  result.shenSha = getShenSha2(result);
  return result;
};
var getJinKouJue = (date, diFen) => {
  const dateInfo = getDateByObj(date);
  return getJinKouJueByDateInfo(dateInfo, diFen);
};

// src/index.ts
var getLiuRenByDate = (time) => {
  const date = getDateByObj(time);
  const riGan = date.bazi.split(" ")[2].substring(0, 1);
  const tianDiPan = getTianDiPan(date);
  const siKe = getSiKe(date, tianDiPan);
  const dunGan = getDunGan(date, tianDiPan);
  const jianChu = getJianChu(date, tianDiPan);
  const sanChuan = fillSanChuan(getSanChuan(siKe, tianDiPan), tianDiPan, dunGan, riGan);
  const shenSha = getShenSha(date);
  const yinYangGuiRen = getYinYangGuiRen(date, tianDiPan);
  const chuJian = getChuJian(date);
  const fuJian = getFuJian(date);
  const result = {
    dateInfo: date,
    tiandipan: tianDiPan,
    siKe,
    dunGan,
    chuJian,
    fuJian,
    jianChu,
    sanChuan,
    shenSha,
    yinYangGuiRen
  };
  return result;
};
var getLiuRenBySiZhu = (year, month, day, hour) => {
  const date = getDateBySiZhu(year, month, day, hour);
  const riGan = date.bazi.split(" ")[2].substring(0, 1);
  const tianDiPan = getTianDiPan(date);
  const siKe = getSiKe(date, tianDiPan);
  const dunGan = getDunGan(date, tianDiPan);
  const jianChu = getJianChu(date, tianDiPan);
  const sanChuan = fillSanChuan(getSanChuan(siKe, tianDiPan), tianDiPan, dunGan, riGan);
  const shenSha = getShenSha(date);
  const yinYangGuiRen = getYinYangGuiRen(date, tianDiPan);
  const chuJian = getChuJian(date);
  const fuJian = getFuJian(date);
  const result = {
    dateInfo: date,
    tiandipan: tianDiPan,
    siKe,
    dunGan,
    chuJian,
    fuJian,
    jianChu,
    sanChuan,
    shenSha,
    yinYangGuiRen
  };
  return result;
};
var getNianMing = (time, gender) => {
  const date = getDateByObj(time);
  const year = date.bazi.split(" ")[0];
  const yearNumber = time.getFullYear();
  const nowNumber = (/* @__PURE__ */ new Date()).getFullYear();
  const age = nowNumber - yearNumber;
  let luNian = "";
  const genderString = gender == 1 ? "\u7537" : "\u5973";
  if (gender == 1) {
    const startYear = sixtyJiaZi_default.indexOf("\u4E19\u5BC5");
    const startIndex = startYear + age;
    luNian = sixtyJiaZi_default[startIndex % 60];
  } else {
    const startYear = sixtyJiaZi_default.indexOf("\u58EC\u7533");
    let startIndex = startYear - age;
    if (startIndex < 0) {
      startIndex = 60 + startIndex;
    }
    luNian = sixtyJiaZi_default[startIndex % 60];
  }
  const result = {
    year,
    gender: genderString,
    luNian
  };
  return result;
};
var getJinKouJueByDate = (time, diFen) => {
  const dateInfo = getDateByObj(time);
  return getJinKouJueByDateInfo(dateInfo, diFen);
};
var getJinKouJueBySiZhu = (year, month, day, hour, diFen) => {
  const dateInfo = getDateBySiZhu(year, month, day, hour);
  return getJinKouJueByDateInfo(dateInfo, diFen);
};
export {
  fillSanChuan,
  getChuJian,
  getDateByObj,
  getDateBySiZhu,
  getDunGan,
  getFuJian,
  getGanZhi2Relation,
  getGanZhi2WuXing,
  getGongIndex,
  getGuiShen,
  getJianChu,
  getJiangShen,
  getJinKouJue,
  getJinKouJueByDate,
  getJinKouJueByDateInfo,
  getJinKouJueBySiZhu,
  getLiuQin,
  getLiuRenByDate,
  getLiuRenBySiZhu,
  getNianMing,
  getSanChuan,
  getShangShen,
  getShenGanZhi,
  getShenSha,
  getSiKe,
  getTianDiPan,
  getTianJiang,
  getWangXiangXiuQiu,
  getWuShuDun,
  getWuXing,
  getXiaShen,
  getYinYangGuiRen,
  getYueLingWuXing
};
